.github/workflows/sims.yml

```diff
@@ -121,27 +121,3 @@ jobs:
         run: |
           make test-sim-multi-seed-short
         if: "env.GIT_DIFF != ''"
-
-  test-liveness:
```

Comment:
was this failing?

Reply:
yes, but I think it's failing for process reasons?  

Basically, it is an attempt to backport and the original v4 branch had a different test.  I could take the .github folder from v4 and get rid of the diff, though.

Reply:
Another way to say this: v4 ci isn't maintained.